### ISSUE TITLE
formatting failed

### ISSUE THREAD

**Issue by HaoHao-Ting** [affenwiesel/matlab-formatter-vscode#55]
VSCode Version 1.45.1
Python Version 3.8.2
Matlab Version 2019 b
I have installed Chinese (Simplified) Language Pack for Visual Studio Code in my VScode. I use Chinese in my PC.
When I try to format the m-file in VScode, it showed me that 

[](url)
[Snipaste of Error](https://tvax2.sinaimg.cn/large/89b34668ly1geu3y14mifj20ea06ut8r.jpg)

I tried UTF-8 in the file. It also doesn't work
Thanks a lot.

**Comment by affenwiesel** [comment]
Two things:

-  I won't click any external links!

- After lots of failed attempts and because of my inability to validate the result, I will no longer try to fix anything concerning chinese letters.

**Comment by WhiskyChoy** [comment]
@affenwiesel Thanks a lot for your work! The problem @HaoHao-Ting shows is actually not caused by the encoding problem. I've checked the code and there aren't such problems. I think it's because of the "matlab-formatter.pythonPath" configuration. In the code file "extension.js" there is no space separator between this config item and the parameters behind. What @HaoHao-Ting shows in the picture is a sequence of Mojibake and I guess the meaning is "no such command xxx" as the command "some_path_to/python.exe" stick with the parameters after it without any space separator. I've created a pull request and I hope it will solve the problem. Thanks again!
